conans/model/manifest.py

```diff
@@ -108,17 +98,27 @@ def create(cls, folder, exports_sources_folder=None):
         from disk, and capturing current time
         """
         files, _ = gather_files(folder)
+        # The folders symlinks are discarded for the manifest
         for f in (PACKAGE_TGZ_NAME, EXPORT_TGZ_NAME, CONAN_MANIFEST, EXPORT_SOURCES_TGZ_NAME):
             files.pop(f, None)
 
         file_dict = {}
         for name, filepath in files.items():
-            file_dict[name] = md5sum(filepath)
+            if not os.path.exists(filepath):  # Broken symlink
+                # FIXME: What to hash? the path?
```

Comment:
Maybe hashing an empty string is enough, the hash already contains the file path:
```python
 s = ["%s: %s" % (f, fmd5) for f, fmd5 in sorted(self.file_sums.items())]
        s.append("")
        return md5("\n".join(s))```

Reply:
we definitely shouldn't hash a symlink target, as it may change (e.g. symlink to `/usr/bin/perl` will be different for different distros), or it may be even infinite (symlinks to files like `/dev/urandom` or `/dev/zero`).
I would just hash the path itself to ensure symlink stays the same.

Reply:
I've hashed the path where the symlink is pointing, after all, that's the content of the file, right? No matter if it is broken or not.